app/lib/services/connect.ts

```diff
@@ -287,8 +289,9 @@ async function login(credentials: ICredentials, isFromWebView = false): Promise<
 			roles: result.me.roles,
 			avatarETag: result.me.avatarETag,
 			isFromWebView,
-			showMessageInMainThread: result.me.settings?.preferences?.showMessageInMainThread ?? true,
-			enableMessageParserEarlyAdoption
+			showMessageInMainThread,
+			enableMessageParserEarlyAdoption,
+			alsoSendThreadToChannel: result.me.settings?.preferences?.alsoSendThreadToChannel
```

Comment:
This field is not reactive.
If you change it on other client, it's not reflecting the change on mobile.

Reply:
Fixed